tests/cases/fourslash/refactorConvertExport_onlyValidIdentifiers.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////*[| |]*/ /** x */ export default 1;
+
+// @Filename: /b.js
+/////*[| |]*/ /** x */ export default (1);
+
+// @Filename: /c.js
+/////*[| |]*/ /** x */ export default x;
+
+goTo.eachRange(r => {
+    goTo.selectRange(r);
```

Comment:
interesting, do you have to select each range? I thought having the cursor there would be enough.

Reply:
But there are three ranges in three files...  In any case, existing cases use markers, which I didn't want to mix with a comment.

Reply:
The cursor there alone is not _supposed_ to be enough unless invoked by the refactor command, because I thought it would be too annoying. See https://github.com/microsoft/TypeScript/blob/master/tests/cases/fourslash/refactorConvertExportForTriggerReason.ts

Reply:
OK. Some refactors are offered in cases other than explicit invoking of the refactor command, right? I always forget about that.